test/parallel/test-http-write-head.js

```diff
@@ -76,3 +76,25 @@ function runTest() {
     response.resume();
   }));
 }
+
+{
+  const server = http.createServer((req, res) => {
+    res.writeHead(200, [ 'test', '1' ]);
+    assert.throws(() => res.writeHead(200, [ 'test2', '2' ]), {
+      code: 'ERR_HTTP_HEADERS_SENT',
+      name: 'Error',
+      message: 'Cannot write headers after they are sent to the client'
```

Comment:
It's better not the check for exact error messages, since they might change. Can you remove this line?